samples/sensor/lis2dh/prj.conf

```diff
@@ -1,4 +1,16 @@
+#Debugging
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_CONSOLE=y
+CONFIG_LOG=y
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_I2C=y
+
+CONFIG_SPI=y
 CONFIG_SENSOR=y
 CONFIG_LIS2DH=y
+
+#Below will enable trigger, not working at the moment
```

Comment:
Please remove the whole section if it's not working, or the commented out line at the very least